project.clj

```diff
@@ -9,18 +9,19 @@
                  ^:source-dep [cider/orchard "0.4.0"]
                  ^:source-dep [thunknyc/profile "0.5.2"]
                  ^:source-dep [mvxcvi/puget "1.1.0"]
-                 ^:source-dep [fipp "0.6.15"]
+                 ^:source-dep [fipp "0.6.15"]; can be removed in unresolved-tree mode
                  ^:source-dep [compliment "0.3.8"]
                  ^:source-dep [cljs-tooling "0.3.1"]
                  ^:source-dep [cljfmt "0.6.4" :exclusions [org.clojure/clojurescript]]
-                 ;; Not used directly in cider-nrepl, but needed because of tools.namespace
-                 ;; and the way MrAnderson processes dependencies
-                 ;; See https://github.com/clojure-emacs/cider/issues/2176 for details
-                 ^:source-dep [org.clojure/java.classpath "0.3.0"]
                  ^:source-dep [org.clojure/tools.namespace "0.3.0-alpha4"]
                  ^:source-dep [org.clojure/tools.trace "0.7.10"]
                  ^:source-dep [org.clojure/tools.reader "1.2.2"]]
-  :plugins [[thomasa/mranderson "0.4.9"]]
+  :plugins [[thomasa/mranderson "0.5.0"]]
+  :mranderson {:project-prefix "cider.nrepl.inlined-deps"
+               :overrides       {[mvxcvi/puget fipp] [fipp "0.6.15"]}; only takes effect in unresolved-tree mode
+               :expositions     [[mvxcvi/puget fipp]]; only takes effect unresolved-tree mode
+               :unresolved-tree false}
```

Comment:
Unrelated comment - why is this needed? Wasn't `:resolved-tree` the default anyways?

Reply:
it is not really needed. but for now i left it like that with the above options that are not needed in resolved tree mode either to make it easier to switch to unresolved tree mode in the near future if the need arises. if not everything apart from the project prefix can be removed.

Reply:
My point was that I thought the default would be `:resolved-deps true` and its opposite would be `:resolved-tree false`. Might have been a bit better if this was `:mode :(un)resolved-tree`, but as long as it gets the job done anything goes.